File: solutions/julia/pangram/1/pangram.jl
"""
    ispangram(input)

Return `true` if `input` contains every alphabetic character (case insensitive).

"""
function ispangram(input::String)::Bool
    str = input .|> Base.Unicode.lowercase
    for char in "abcdefghijklmnopqrstuvwxyz"
        if occursin(char, str)
            continue
        end
        return false
    end
    return true
end

File: solutions/ocaml/pangram/1/pangram.ml
let is_pangram s =
  let s = String.lowercase_ascii s in
  String.for_all (String.contains s) "abcdefghijklmnopqrstuvwxyz"

(* open Base *)

(* let is_pangram s = *)
  (* let s = String.lowercase s in *)
  (* String.for_all "abcdefghijklmnopqrstuvwxyz" ~f:(String.contains s) *)

